pipelines/ocw/pipeline-theme-assets.yml

```diff
@@ -0,0 +1,57 @@
+---
+resource_types:
+  - name: rclone
+    type: docker-image
+    source:
+      repository: mitodl/concourse-rclone-resource
+      tag: latest
```

Comment:
Is it normal that we have to define this `rclone` `resource type` in most every pipeline?

Reply:
Yes, because it's not a default resource for the installation (yet). That's how you tell Concourse to pull down the resource container to use for the pipeline build